test/database/meson.build

```diff
@@ -0,0 +1,19 @@
+run_command('cp', meson.source_root() + '/src/database/sqlite3/sqlite3.sql', meson.current_build_dir(), check : true)
+run_command('cp', meson.source_root() + '/src/database/sqlite3/sqlite3-upgrade.xml', meson.current_build_dir(), check : true)
+run_command('cp', meson.source_root() + '/src/database/mysql/mysql.sql', meson.current_build_dir(), check : true)
+run_command('cp', meson.source_root() + '/src/database/mysql/mysql-upgrade.xml', meson.current_build_dir(), check : true)
```

Comment:
Probably makes sense to use `configure_file(..., copy: true)` here, as that runs in-process, is cross-platform, and gets updated when the original files are modified.

Reply:
Ideally it would work with something like the previous revision of this PR, by just setting the `workdir: ` of the test so it picks up the files in the source tree...

Reply:
Fixed.